packages/core/src/tracing/cls.js

```diff
@@ -10,39 +10,72 @@ const tracingUtil = require('./tracingUtil');
 const { ENTRY, EXIT, INTERMEDIATE } = require('./constants');
 const hooked = require('./clsHooked');
 const tracingMetrics = require('./metrics');
+
+/**
+ * @typedef {Object} GetFromProperties
+ * @property {string} e
+ * @property {string} h
+ */
+
+/**
+ * @typedef {Function} GetFromFunction
+ * @returns {GetFromProperties}
+ */
+
+/**
+ * @description This object is actually here: /nodejs-sensor/packages/collector/src/pidStore/index.js
+ * @typedef {Object} PIDStore
+ * @property {GetFromFunction} getFrom
+ */
```

Comment:
I explicitly added a comment saying that PIDStore belongs to the collector, which I am not instrumenting here.
Touching something else means that Typescript will look for all dependencies, even though we don't clearly say that we don't want to instrument a folder. This may or may not happen in this case, but from my experience so far, it becomes a rabbit hole very easily.

If we add this interface in the /core area: This is not actually Typescript, so creating a file for an interface means that this file will only exist because of the interface. In a "normal" TS project, such file would be deleted during the build.
Having that said, where would you recommend that the interface is created?